be/src/vec/functions/if.cpp

```diff
@@ -321,23 +321,25 @@ class FunctionIf : public IFunction {
                                                       else_col.type->get_type_id(), call);
     }
 
-    bool execute_for_null_then_else(FunctionContext* context, Block& block,
-                                    const ColumnWithTypeAndName& arg_cond,
-                                    const ColumnWithTypeAndName& arg_then,
-                                    const ColumnWithTypeAndName& arg_else, size_t result,
-                                    size_t input_rows_count, Status& status) const {
+    Status execute_for_null_then_else(FunctionContext* context, Block& block,
+                                      const ColumnWithTypeAndName& arg_cond,
+                                      const ColumnWithTypeAndName& arg_then,
+                                      const ColumnWithTypeAndName& arg_else, size_t result,
+                                      size_t input_rows_count, bool& handled) const {
         bool then_is_null = arg_then.column->only_null();
         bool else_is_null = arg_else.column->only_null();
 
+        handled = false;
```

Comment:
seems no need set hanled false here ？ out the function have already set the `false`

Reply:
I think assigning it a value of false might be safer and will not affect the performance.